lib/parse.js

```diff
@@ -1320,11 +1320,17 @@ function parse($TEXT, options) {
             var type = start.type;
             var name = as_property_name();
             if (type == "name" && !is("punc", ":")) {
+                var createAccessor = function() {
```

Comment:
I do not mind, however this is beyond my current JS abilities. What I have tried (and did not even compile is):

                    a.push(new AST_ObjectGetter({
                        start : start,
                        key   : as_atom_node(),
                        value : embed_tokens(function(){function_(AST_Accessor)}),
                        end   : prev()
                    }));

If you can advise me, I will be glad to implement it.